sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/_tests_/e2e/application-offering-change-request.institutions.controller.createApplicationOfferingChangeRequest.e2e-spec.ts

```diff
@@ -526,6 +526,64 @@ describe("ApplicationOfferingChangeRequestInstitutionsController(e2e)-createAppl
       });
   });
 
+  it("Should throw program is expired error when trying to submit an application offering request for an expired program.", async () => {
+    // Arrange
+    const savedUser = await db.user.save(createFakeUser());
+    // Student has a completed application to the institution.
+    const application = await saveFakeApplicationDisbursements(
+      db.dataSource,
+      { institutionLocation: collegeFLocation },
+      {
+        applicationStatus: ApplicationStatus.Completed,
+      },
+    );
+
+    await db.student.save(application.student);
+    await db.application.save(application);
+
+    // New offering with full time intensity.
+    const newOffering = createFakeEducationProgramOffering(
+      {
+        auditUser: savedUser,
+        institutionLocation: collegeFLocation,
+      },
+      {
+        initialValues: {
+          studyStartDate: addDays(
+            5,
+            application.programYear.startDate,
+          ).toISOString(),
```

Comment:
Minor, `studyStartDate` and `studyEndDate` are date only and the `toISOString` will include time also.
We can use the `getISODateOnlyString` instead.